webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -33,14 +35,24 @@ export function mount(component, selector, data) {
     StorageContainer: {
       type: StorageContainer,
       markup: <StorageContainer store={store} data={data} />
+    },
+    PersonalAccessToken: {
+      type: PersonalAccessToken,
+      markup: <PersonalAccessToken data={data} />
```

Comment:
nitpick: can use {...data} instead to save the props.data.url and just props.url